fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCNormalizedImpl.java

```diff
@@ -410,11 +413,35 @@ public boolean isDeleteSupported() {
                 searchContext.setLastPageNumber(lastPageNumber);
                 
                  
-                if (searchResultCount > 0) {
+                // For _summary=count, we don't need to return any resource 
+                if (searchResultCount > 0 && 
+                        !(searchContext.getSummaryParameter() != null 
+                        && searchContext.getSummaryParameter().equals(SearchConstants.SUMMARY_COUNT))) {
                     query = queryBuilder.buildQuery(resourceType, searchContext);
                     
                     List<String> elements = searchContext.getElementsParameters();
                     
+                    //Only consider _summary if _elements parameter is empty
+                    if (elements == null) {
+                        Set<String> summaryElements = null;
+                        String summary = searchContext.getSummaryParameter();
+                        if (summary != null) {
+                            if (summary.equals(SearchConstants.SUMMARY_TRUE)) {
+                                summaryElements = JsonSupport.getSummaryElementNames(resourceType);
+                            } else if (summary.equals(SearchConstants.SUMMARY_TEXT)) {
+                                summaryElements = SearchUtil.getSummaryTextElementNames(resourceType);
+                            } else if (summary.equals(SearchConstants.SUMMARY_DATA)) {
+                                //summaryElements = SearchUtil.getSummaryDataElementNames(resourceType);
```

Comment:
why is this line commented out?  should it be removed?

Reply:
removed.